_posts/2022-11-02-relicensing.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License][GPLv2] (GPL v2+) to the more permissive [Lesser GNU
```

Comment:
todo - don't call this permissive

Reply:
Why not?

Reply:
permissive is a very specific category under licensing, imho just bringing it up opens you up to further scrutiny. It may be that this is the only word we can use here, but I'd like to give it some thought ahead of using such specific language.

Reply:
ok, if this is too loaded then we don't want to stir up controversy, and of course not overclaim.